src/atcore.cpp

```diff
@@ -324,3 +324,31 @@ void AtCore::detectFirmware()
     connect(serial(), &SerialLayer::receivedCommand, this, &AtCore::findFirmware);
     requestFirmware();
 }
+/*~~~~~Control Slots ~~~~~~~~*/
+void AtCore::home()
+{
+    pushCommand(GCode::toCommand(GCode::G28));
+}
+
+void AtCore::home(QString axis)
+{
+    axis.replace(QChar::fromLatin1('X'), QStringLiteral("X0"));
+    axis.replace(QChar::fromLatin1('Y'), QStringLiteral("Y0"));
+    axis.replace(QChar::fromLatin1('Z'), QStringLiteral("Z0"));
+    pushCommand(GCode::toCommand(GCode::G28, axis));
+}
```

Comment:
This function can use other function called `move(axis, pos)`, in that way we can have
a generic move function and a home function that is very simple, just calling a move